components/src/molecules/file-upload/FileUpload.js

```diff
@@ -0,0 +1,104 @@
+import AButton from '../../atoms/button/Button.vue'
+import ALabel from '../../atoms/label/Label.vue'
+
+export default {
+  components: {
+    AButton,
+    ALabel
+  },
+  props: {
+    /**
+     * File input id
+     */
+    inputId: {
+      type: String,
+      default: 'file'
+    },
+    /**
+     * File input name
+     */
+    inputName: {
+      type: String,
+      default: 'file'
+    },
+    /**
+     * Top title
+     */
+    title: {
+      type: String,
+      default: 'Your file'
+    },
+    /**
+     * Button text
+     */
+    buttonText: {
+      type: String,
+      default: 'Upload a file'
```

Comment:
we can't pass default values for string props b/c translations, move to the story, please change for each occurs

Reply:
Ok, done